spec.html

```diff
@@ -42262,7 +42262,7 @@ <h1>Properties of Map Instances</h1>
 
     <emu-clause id="sec-map-iterator-objects">
       <h1>Map Iterator Objects</h1>
-      <p>A Map Iterator is an object, that represents a specific iteration over some specific Map instance object. There is not a named constructor for Map Iterator objects. Instead, map iterator objects are created by calling certain methods of Map instance objects.</p>
+      <p>A <dfn variants="Map iterators,Map iterator object,Map iterator objects">Map iterator</dfn> is an object, that represents a specific iteration over some specific Map instance object. There is not a named constructor for Map iterator objects. Instead, map iterator objects are created by calling certain methods of Map instance objects.</p>
```

Comment:
You did `s/object, that/object that/` on the other places this boilerplate appears; probably should do here as well.